docs/user/agent/minimal-iso.md

```diff
@@ -1,11 +1,9 @@
-# External platform configurations
-
-This document provides comprehensive details for configuring the external platform using the agent-based installer for OpenShift. It covers the minimum version requirement, 
-configuration options, and the generation of minimal ISOs for the specified platform.
+# Minimal ISO configurations
 
+The agent-based installer will generate a minimal ISO (without the rootfs) at all times when the platform type is `External`. In addition, it can be generated for other platform types when configured.  This document provides comprehensive details for configuring minimal ISOs using the agent-based installer for OpenShift. It covers the minimum version requirement, configuration options, and the method of retrieving the rootfs.
 
 ## Minimum OpenShift Version
-The agent-based installer for OpenShift requires a minimum OpenShift version of 4.14 to support external platforms.
+The agent-based installer for OpenShift requires a minimum OpenShift version of 4.14 to support external platforms. To support minimal ISOs for other platform types, a minimum Openshift version of 4.18 is required.
```

Comment:
Its not necessary since adding the new agent-config.yaml field to versions prior to 4.18 will result in
`level=fatal msg=failed to fetch Agent Installer ISO: failed to load asset "Agent Config": failed to unmarshal agent-config.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field "minimalISO"`